web-app/js/portal/search/FacetedSearchResultsDataView.js

```diff
@@ -57,6 +57,10 @@ Portal.search.FacetedSearchResultsDataView = Ext.extend(Ext.DataView, {
                 },
                 getIconUrl: function(values) {
                     return Ext.ux.Ajax.constructProxyUrl(this.getGeonetworkImageUrl(values.iconSourceUuid));
+                },
+                getTrimmedTitle: function(values) {
+                    var title = values.title;
+                    return String.format("<h3 title=\"{0}\">{1}</h3>", title, Ext.util.Format.ellipsis(title, 180, true));
```

Comment:
I thought it would be nice to have all HTML tags inside `_getHtmlContent` and make the function `getTrimmedTitle` do just what the name suggests. Does that make sense?